### DISCUSSION TITLE
Expression for pitch?

### DISCUSSION THREAD

**Discussion by devolod** [maplibre/maplibre-gl-js#4207] [Q&A] [answered]
Hi All! 
I need increase width of line for `pitch` > 0. 
Is there `Expression` or `Camera expression` for `paint.line-width` that depends on `pitch`?

**Comment by HarelM** [comment]
I think you can do it programmatically, but not using any expression.

**Reply by devolod** [reply]
Could you provide minimum example?

**Reply by HarelM** [reply, marked as answer]
Something like this example:
https://maplibre.org/maplibre-gl-js/docs/examples/color-switcher/

But using `map.on('move', () => ...)` or `map.on('pitch', () => ...)`

**Reply by devolod** [reply]
Thanks!

**Reply by pm0u** [reply]
We are running into this issue as well in similar circumstances (expecting parity). Thanks for the explanation. Just to try to understand this further, it sounds like this particular feature isn't planned for maplibre? Is this a technical limitation/decision or are there legal concerns around mirroring mapbox spec?

**Reply by HarelM** [reply]
The best place to discuss implementation and spec changes is the style spec repo, this issue in particular: 
https://github.com/maplibre/maplibre-style-spec/issues/346
